### DISCUSSION TITLE
How to add more font in Print Designer with out modify the core code.

### DISCUSSION THREAD

**Discussion by darkmanst** [frappe/print_designer#198] [Q&A]
How to add more font in Print Designer with out modify the core code.

**Comment by ameerberg** [comment]
+1 to this. 
I'm using Arabic language and in need of using my custom fonts.

**Comment by mask2082** [comment]
I was wondering if adding the line below to print_designer/public/js/print_designer/defaultObjects.js is enough to add a Lexend Deca font from Google fonts to Print Designer? If so how can it be done, please?

```
LexendDeca: [
        [100, 200, 300, 400, 500, 600, 700, 800, 900], []],
```

**Reply by maharshivpatel** [reply]
@mask2082 yes that is enough to add font if it is available in google fonts.

raise a PR and i will merge it.

**Reply by mask2082** [reply]
@maharshivpatel Thanks, all done.
